parquet-column/src/main/java/org/apache/parquet/schema/Type.java

```diff
@@ -140,8 +147,14 @@ public Type(String name, Repetition repetition, OriginalType originalType) {
    */
   Type(String name, Repetition repetition, OriginalType originalType, ID id) {
     super();
-    this.name = checkNotNull(name, "name");
+
+    name = checkNotNull(name, "name");
+    if (name.charAt(0) == '`') {
+      name = name.substring(1, name.length() - 1).replace("``", "`");
+    }
```

Comment:
we should check that the last character is also a quote.
We should factor out the logic with MessageTypeParser.Tokenizer.getName()